apis/v1alpha1/httproute_types.go

```diff
@@ -608,7 +630,7 @@ type HTTPRouteForwardTo struct {
 	// in HTTPRouteRule.)
 	//
 	// +optional
-	// +kubebuilder:validation:MaxItems=16
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
I think we should only tighten validation with new API versions. If we want to do this I think it should be part of v1alpha2.

Reply:
@robscott done.